sos/plugins/python.py

```diff
@@ -10,6 +10,9 @@
 
 from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
 import sos.policies
+import json
+import hashlib
+from pathlib import Path
```

Comment:
This fails on `python2` by default. Either use a `py2` compatible library, or wait until we drop `py2` support in sosreport.